lib/ansible/modules/network/ios/ios_acl.py

```diff
@@ -0,0 +1,1430 @@
+            aces:
+              - grant: deny
+                protocol_options:
+                  tcp:
+                    ack: true
+                source:
+                  address: 198.51.100.0
+                  wildcard_bits: 0.0.0.255
+                destination:
+                  address: 198.51.101.0
+                  wildcard_bits: 0.0.0.255
+                  port_protocol:
+                    eq: telnet
+                tos:
+                  service_value: 12
+              - grant: deny
+                protocol_options:
+                  tcp:
+                    ack: true
+                source:
+                  address: 192.0.3.0
+                  wildcard_bits: 0.0.0.255
+                destination:
+                  address: 192.0.4.0
+                  wildcard_bits: 0.0.0.255
+                  port_protocol:
+                    eq: www
+                dscp: ef
+                ttl:
+                  lt: 20
+      - afi: ipv6
+        acls:
+          - name: R1_TRAFFIC
+            aces:
+              - grant: deny
+                protocol_options:
+                  tcp:
+                    ack: true
+                source:
+                  any: true
+                  port_protocol:
+                    eq: www
+                destination:
+                  any: true
+                  port_protocol:
+                    eq: telnet
+                dscp: af11
+    state: merged
+
+# Commands fired:
+# ---------------
+#
+# - ip access-list standard std_acl
+# - deny 192.168.1.200
+# - deny 192.168.2.0 0.0.0.255
+# - ip access-list extended 110
+# - no 10
+# - 10 deny icmp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 traceroute dscp ef ttl eq 10
+# - deny tcp host 198.51.100.0 host 198.51.110.0 eq telnet ack
+# - ip access-list extended test
+# - deny tcp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 eq www fin option traceroute ttl eq 10
+# - ip access-list extended 123
+# - deny tcp 198.51.100.0 0.0.0.255 198.51.101.0 0.0.0.255 eq telnet ack tos 12
+# - deny tcp 192.0.3.0 0.0.0.255 192.0.4.0 0.0.0.255 eq www ack dscp ef ttl lt 20
+# - ipv6 access-list R1_TRAFFIC
+# - deny tcp any eq www any eq telnet ack dscp af11
+
+# After state:
+# ------------
+#
+# vios#sh access-lists
+# Standard IP access list std_acl
+#    10 deny   192.168.1.200
+#    20 deny   192.168.2.0, wildcard bits 0.0.0.255
+# Extended IP access list 110
+#    10 deny icmp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 traceroute dscp ef ttl eq 10
+#    20 deny tcp host 198.51.100.0 host 198.51.110.0 eq telnet ack
+# Extended IP access list 123
+#    10 deny tcp 198.51.100.0 0.0.0.255 198.51.101.0 0.0.0.255 eq telnet ack tos 12
+#    20 deny tcp 192.0.3.0 0.0.0.255 192.0.4.0 0.0.0.255 eq www ack dscp ef ttl lt 20
+# Extended IP access list test
+#    10 deny tcp 192.0.2.0 0.0.0.255 192.0.3.0 0.0.0.255 eq www fin option traceroute ttl eq 10
+# IPv6 access list R1_TRAFFIC
+#    deny tcp any eq www any eq telnet ack dscp af11 sequence 10
+
```

Comment:
Please add an example to demonstrate updating an existing ACE using merged.

Reply:
+1

Reply:
I see it's already for `ip access-list extended 110` sequence 10 ace, where in case of update merge is negating the existing ace value and applying the new want ace change.